student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_spec'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Since the Student class inherits from the Person class I'd suggest adding a default value to the `name` and `parent_permission` parameters just like you did for the super class.